recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set name = "rubin-env" %}
-{% set version = "9.0.0" %}
-{% set build_number = 11 %}
+{% set version = "9.0.1" %}
+{% set build_number = 12 %}
```

Comment:
If you are incrementing the patch version you need to reset the build number to 0.

Reply:
Ok, does that looks right now? (first time I am doing this)